src/rdb/tests/rdbt.c

```diff
@@ -40,28 +48,71 @@ help_hdlr(int argc, char *argv[])
 	printf("\
 usage: rdbt COMMAND [OPTIONS]\n\
 commands:\n\
-  init	init a replica\n\
-  fini	finalize a replica\n\
-  test	invoke tests on a replica\n\
-  help	print this message and exit\n");
+  init		init a replica\n\
+  create	create KV stores (on discovered leader)\n\
+  test		invoke tests on a specified replica rank\n\
+  test-multi	invoke tests (on discovered leader)\n\
+  destroy	destroy KV stores (on discovered leader)\n\
+  fini		finalize a replica\n\
+  help		print this message and exit\n");
 	printf("\
 init options:\n\
   --group=GROUP	server group \n\
   --rank=RANK	rank to initialize (0)\n\
   --replicas=N	number of replicas (1)\n\
   --uuid=UUID	rdb UUID\n");
 	printf("\
-fini options:\n\
+create, test-multi, destroy options:\n\
   --group=GROUP	server group \n\
-  --rank=RANK	rank to initialize (0)\n");
+  --replicas=N	number of replicas (1)\n\
+  --nranks=R	number of server ranks (1)\n");
 	printf("\
 test options:\n\
   --group=GROUP	server group \n\
   --rank=RANK	rank to invoke tests on (0)\n\
   --update	update (otherwise verify)\n");
+	printf("\
+fini options:\n\
+  --group=GROUP	server group \n\
+  --rank=RANK	rank to finalize (0)\n");
 	return 0;
 }
 
+/**** Common utility functions for multiple tests. */
+
+static int
+multi_tests_common_parse(int argc, char *argv[])
+{
+	struct option		options[] = {
+		{"group",	required_argument,	NULL,	'g'},
+		{"nranks",	required_argument,	NULL,	'n'},
+		{"replicas",	required_argument,	NULL,	'R'},
+		{NULL,		0,			NULL,	0}
+	};
+	int			rc;
+
+	while ((rc = getopt_long(argc, argv, "", options, NULL)) != -1) {
+		switch (rc) {
+		case 'g':
+			group_id = optarg;
+			break;
+		case 'n':
+			g_nranks = atoi(optarg);
+			break;
+		case 'R':
+			g_nreps = atoi(optarg);
+			break;
+		default:
+			return 2;
+		}
+	}
+
+	D_ASSERTF((g_nreps < g_nranks), "g_nreps=%u <= g_nranks=%u\n",
```

Comment:
(style)  spaces required around that '=' (ctx:VxO)
(style)  spaces required around that '=' (ctx:VxO)